backend/src/launcher.py

```diff
@@ -5,6 +5,7 @@
 
 database_directory = "../data/games.json"
 games_directory = "/home/pi/ConsoleGames"
+cover_image_directory = "../data/cover_images"
```

Comment:
should let the user know that images will need to be stored in the ../data/cover_images

Reply:
On it